featuretools/utils/cli_utils.py

```diff
@@ -40,3 +80,7 @@ def get_installed_packages():
     for d in pkg_resources.working_set:
         installed_packages[d.project_name] = d.version
     return installed_packages
+
+
+def get_featuretools_root():
```

Comment:
if we move this, we should also move the test to existing `test_cli_utils.py` file

Reply:
Fixed.